src/components/cc-input-date/cc-input-date.test.js

```diff
@@ -0,0 +1,580 @@
+        });
+      }
+      for (const position of [8, 9, 10]) {
+        it(`should shift up date day when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowUp', '2023-08-01T19:11:12.259Z');
+        });
+      }
+      for (const position of [11, 12, 13]) {
+        it(`should shift up date hour when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowUp', '2023-07-31T20:11:12.259Z');
+        });
+      }
+      for (const position of [14, 15, 16]) {
+        it(`should shift up date minute when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowUp', '2023-07-31T19:12:12.259Z');
+        });
+      }
+      for (const position of [17, 18, 19]) {
+        it(`should shift up date second when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowUp', '2023-07-31T19:11:13.259Z');
+        });
+      }
+    });
+
+    describe('ArrowDown', () => {
+      for (const position of [0, 1, 2, 3, 4]) {
+        it(`should shift down date year when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowDown', '2022-07-31T19:11:12.259Z');
+        });
+      }
+      for (const position of [5, 6, 7]) {
+        it(`should shift down date month when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowDown', '2023-06-30T19:11:12.259Z');
+        });
+      }
+      for (const position of [8, 9, 10]) {
+        it(`should shift down date day when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowDown', '2023-07-30T19:11:12.259Z');
+        });
+      }
+      for (const position of [11, 12, 13]) {
+        it(`should shift down date hour when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowDown', '2023-07-31T18:11:12.259Z');
+        });
+      }
+      for (const position of [14, 15, 16]) {
+        it(`should shift down date minute when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowDown', '2023-07-31T19:10:12.259Z');
+        });
+      }
+      for (const position of [17, 18, 19]) {
+        it(`should shift down date second when cursor position is ${position}`, async () => {
+          await checkAfterShift('2023-07-31T19:11:12.259Z', position, 'ArrowDown', '2023-07-31T19:11:11.259Z');
+        });
+      }
+    });
+  });
+
+  describe(`accessibility`, function () {
+    testAccessibility(getStories(storiesModule));
+  });
```

Comment:
**note:** ah yes, I think we need to change the `describe` used in the `accessibility` test helper to avoid redundant info.

Your test displays this if it fails:
```
cc-input-date > accessibility > Story: Required > Accessibility > Mobile: width = 360 height = 640
```

This is not your fault, we need to remove one `describe` layer within the `accessibility` test helper.